lib/jsdom/utils.js

```diff
@@ -169,6 +169,13 @@ exports.Canvas = null;
       exports.Canvas = null;
     }
   } catch (e) {
+    try {
+      require.resolve(moduleName);
+      // the module is installed, but loading it failed
+      console.error("Failed loading " + moduleName, e);
```

Comment:
Alternatively, we could potentially also rethrow here, which would make it even more obvious, however that would be a backwards incompatible change. Let me know if you prefer to rethrow.